pkgs/top-level/node-packages.nix

```diff
@@ -25,6 +25,15 @@ rec {
     "node-protobuf".buildInputs = [ pkgs.protobuf ];
     "rbytes".buildInputs = [ pkgs.openssl ];
 
+    # Remove fsevents dependency because it is OSX-specific.
```

Comment:
Hm yea, it's optional [1] but it shows up as `"fsevents-0.3.5" = self.by-version."fsevents"."0.3.5";` in the npm2nix output.. I can retract this PR and wait for platform support? webpack is not urgent at all, I can just keep a local version around.

[1]
https://github.com/paulmillr/chokidar/blob/master/package.json